### PR TITLE
Add the Chromium-based Edge to supported browsers

### PR DESCRIPTION
**Pull request by zanza00**
Add to readme how to enable chrome extensions, with link to official docs, in the new Edge.

Tested on Microsoft Edge `Version 80.0.361.48 (Official build) (64-bit)` on Mac Os Catalina `10.15.2`

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ GitHub Enterprise is also supported. More info in the options.
 - [**Chrome** extension][link-cws] [<img valign="middle" src="https://img.shields.io/chrome-web-store/v/hlepfoohegkhhmjieoechaddaejaokhf.svg?label=%20">][link-cws]
 - [**Firefox** add-on][link-amo] [<img valign="middle" src="https://img.shields.io/amo/v/refined-github-.svg?label=%20">][link-amo]
 - **Opera** extension: Use [this Opera extension](https://addons.opera.com/en/extensions/details/download-chrome-extension-9/) to install the Chrome version.
+- **Edge** extension: Follow [this guide](https://support.microsoft.com/en-us/help/4538971/microsoft-edge-add-or-remove-extensions) to install the Chrome Version (applicable to New Edge version)
```
Comment (sindresorhus) [inline readme.md]:
```suggestion
- **Edge** extension: Follow [this guide](https://support.microsoft.com/en-us/help/4538971/microsoft-edge-add-or-remove-extensions) to install the Chrome version (only applicable to Chromium-based Edge).
```

**Comment by fregante** [comment]
There’s no guide to be followed, users can just open the Chrome link and click “enable installs from this site” (or whatever the info bar says)

The only change I’d accept is something like “The chrome version also works in Opera (using _this_) and Edge.”

**Comment by zanza00** [comment]
Updated description as suggested :)

**Comment by zanza00** [comment]
Can I merge this myself?

**Pull request merged**